packages/Layout/header/src/NavBar/NavBar.js

```diff
@@ -7,7 +7,7 @@ import { setPosition } from './NavBar.helpers';
 const propTypes = {
   ...Constants.propTypes,
   isVisible: PropTypes.bool.isRequired,
-  onClick: PropTypes.func.isRequired,
+  onClick: PropTypes.func,
```

Comment:
no problem here for just set props not required :)

Reply:
can you add an example in the story , please ? 
with no onClick props : https://github.com/AxaGuilDEv/react-toolkit/tree/master/storybook/storybook/src/packages/Layout/header/src/NavBar